task/08-objects-tasks.js

```diff
@@ -23,7 +23,13 @@
  *    console.log(r.getArea());   // => 200
  */
```

Comment:
There is no the link to the travis-ci successful build

Reply:
https://travis-ci.org/github/Inquisitor1229/js-assignments/branches